src/app/core/admin/admin-menu/admin-menu.component.scss

```diff
@@ -0,0 +1,5 @@
+@use "variables/sizes";
+
+.padding-regular {
```

Comment:
Could we just use the `padding-regular` global class directly in the html? then this wouldn’t be needed
```suggestion

```